repackaging/images.yaml

```diff
@@ -54,7 +54,7 @@ images:
   - name: firecrawl
     type: node
     package: firecrawl-mcp
-    version: 2.0.2
+    version: 3.3.6
```

Comment:
Seems like a big jump.

Reply:
yeah.. thats what we get for not keeping these updated before. looks like the major version bump is because they added versioning to the tools:

```
The server provides versioned endpoints to maintain backward compatibility while supporting the latest Firecrawl features:

    V1 (Legacy): Uses Firecrawl JS 1.29.3 with extended tools
    V2 (Current): Uses Firecrawl JS 3.1.0 with modern API
```